### DISCUSSION TITLE
历史命令自动去重，并能自定义排序

### DISCUSSION THREAD

**Discussion by tonywgx123** [kingToolbox/WindTerm#2325] [Ideas]
建议如下：
1：不想要出现的历史命令可以右键删除、或del键删除，可以crtl、shift键配合使用；
2：排序建议：
    2.1：可按照使用次数排序；
    2.2：按照名称排序；
    2.3：可自定义置顶固定多条历史命令（优先级最高，其它排序不能打乱置顶命令）；

**Comment by robot-dot-win** [comment]
Second this!
